lib/certUtils.js

```diff
@@ -79,11 +78,19 @@ class Certificate {
 	}
 
 	getIssuer() {
-		return this._cert.issuer.typesAndValues[0].value.valueBlock.value;
+		return this.searchForCommonName(this._cert.issuer.typesAndValues);
 	}
 
-	getSerial() {
-		return this._cert.subject.typesAndValues[0].value.valueBlock.value;
+	getSerial(compatability) {
```

Comment:
Could you rename the variable to "compatibility"? Looks like a typo here

Reply:
Oops lol. Done.